cmake/clean_tests.cmake

```diff
@@ -0,0 +1,10 @@
+if (BUILD_TESTING)
+
+    FILE(GLOB OLD_FILES "${CMAKE_CURRENT_BINARY_DIR}/../tests/Head*")
+    SET(TMP_FILE "Head-tmp.txt")
+    LIST(APPEND OLD_FILES ${OLD_FILES} ${TMP_FILE})
+    LIST(REMOVE_DUPLICATES OLD_FILES)
+
+    add_test(CLEAN-TESTS ${CMAKE_COMMAND} -E remove -f ${OLD_FILES})
```

Comment:
see this example taken from [here](https://github.com/5gon12eder/msc-graphstudy/blob/8c450f6723f90ea104db732173597e7f9f709f6a/driver-tests.cmake#L65)

```cmake
add_test(
    NAME systest-driver-integrity-1st
    WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
    COMMAND "${Python3_EXECUTABLE}" -m driver.integrity
            "--configdir=${PROJECT_SOURCE_DIR}/test/config/"
            "--datadir=${PROJECT_BINARY_DIR}/_systest/output/"
            "--bindir=${PROJECT_BINARY_DIR}/"
            --log-level=INFO
)
```